oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -512,7 +526,13 @@ public GoogleCredentials createScoped(Collection<String> newScopes) {
         tokenServerUri,
         serviceAccountUser,
         projectId,
-        quotaProjectId);
+        quotaProjectId,
+        lifetime);
+  }
+
+  /** Clones the service account with a new lifetime value * */
```

Comment:
Please add the full javadoc to make sure it's clear that the lifetime is in seconds.

Reply:
done